tests/retrieval/test_mrr.py

```diff
@@ -28,7 +28,7 @@
     _errors_test_functional_metric_parameters_default,
 )
 from torchmetrics.functional.retrieval.reciprocal_rank import retrieval_reciprocal_rank
-from torchmetrics.retrieval.mean_reciprocal_rank import RetrievalMRR
+from torchmetrics.retrieval.rreciprocal_rank import RetrievalMRR
```

Comment:
Why did you add an `r` at the beginning of the name?

Reply:
Good catch 👍